CHANGELOG.md

```diff
@@ -11,6 +11,56 @@ Some experimental features can be enabled / disabled using [Feature Gates](docs/
 
 ## Unreleased
 
+## 0.10.0 - 2020-09-24
+
+Includes all the bug fixes from [0.9.1], [0.9.2] and [0.9.3].
+
+Starting with Antrea 0.10.0, K8s version >= 1.16 is required.
+
+### Added
+
+- Add Antrea NetworkPolicy CRD API to define namespaced security policies with support for priorities. ([#1117](https://github.com/vmware-tanzu/antrea/pull/1117) [#1194](https://github.com/vmware-tanzu/antrea/pull/1194), [@Dyanngg] [@abhiraut]) [Alpha - Feature Gate: `AntreaPolicy`]
+   * The `ClusterNetworkPolicy` Feature Gate has been removed, `AntreaPolicy` is used for both Antrea NetworkPolicies and ClusterNetworkPolicies
+   * Refer to the [Antrea Policy CRDs documentation] for information
+- Add "v1alpha1.stats.antrea.tanzu.vmware.com" API to query traffic statistics about NetworkPolicies (number of sessions / packets / bytes which are allowed or denied). ([#1172](https://github.com/vmware-tanzu/antrea/pull/1172) [#1221](https://github.com/vmware-tanzu/antrea/pull/1221) [#1140](https://github.com/vmware-tanzu/antrea/pull/1140), [@tnqn] [@weiqiangt]) [Alpha - Feature Gate: `NetworkPolicyStats`]
+   * The stats are aggregated from each Antrea Agent using an internal API in "controlplane.antrea.tanzu.vmware.com"
+- Support for ExternalEntity: rules in Antrea policies can select labelled non-Pod enpoints (e.g. VMs) which are represented by ExternalEntity CRD resources; these entities are included in the AddressGroups computed by the Controller and processed by the Agents. ([#1084](https://github.com/vmware-tanzu/antrea/pull/1084), [@Dyanngg] [@suwang48404])
+- Support for querying the list of NetworkPolicies which are applied to a specific Pod, or which select a specific Pod in an ingress / egress rule. ([#1116](https://github.com/vmware-tanzu/antrea/pull/1116), [@jakesokol1] [@antoninbas]) [Alpha]
+   * New "/endpoint" API endpoint in Antrea Controller - API may change in future releases
+   * New "antctl query endpoint" command
+- Add Prometheus metrics for the connection tracking table (max size, total number of connections, total number of connections installed by Antrea) when `FlowExporter` is enabled. ([#1232](https://github.com/vmware-tanzu/antrea/pull/1232), [@dreamtalen])
+- Configure access to Antrea NetworkPolicy and ClusterNetworkPolicy APIs for [default cluster roles] (admin/ edit / view) using [aggregated ClusterRoles]. ([#1206](https://github.com/vmware-tanzu/antrea/pull/1206), [@abhiraut])
```

Comment:
nit: spaces are not consistent between admin, edit, and view. So do the following line.